src/include/rbd/librbd.h

```diff
@@ -1436,15 +1505,18 @@ CEPH_RBD_API void rbd_config_image_list_cleanup(rbd_config_option_t *options,
 CEPH_RBD_API int rbd_group_image_add(rados_ioctx_t group_p,
                                      const char *group_name,
                                      rados_ioctx_t image_p,
-                                     const char *image_name);
```

Comment:
Hi all. I was pointed at this PR due to a PR filed on go-ceph project. In particular I noticed changes to existing public API. Here's one example. I do not think that changing an existing public api is going to be acceptable. IMO you need a new set of API functions if you want to add a new argument.  At least that's been my experience with previous API changes, am I correct @idryomov ?

Reply:
@phlogistonjohn absolutely. This is WIP so we are taking that luxury to see what fits right. The final version of the PR will have new v2 API's if at all any changes are needed to the signatures.

Reply:
Excellent news, thanks!